javascript/src/main/pages/Home/Home.js

```diff
@@ -37,7 +37,15 @@ const Home = () => {
             <div className="text-left">
                 <h5>Welcome to the UCSB Courses Search App!</h5>
                 <BasicCourseSearchForm setCourseJSON={setCourseJSON} fetchJSON={fetchBasicCourseJSON} />
-                <Button><CSVLink style={{color: "white"}} headers={courseHeaders} data={courseJSON.classes} filename = {"CourseTable.csv"}>Download CSV</CSVLink></Button>
+                <Button style={{margin: "1rem 0"}}>
+                    <CSVLink
+                        style={{color: "white"}}
+                        headers={courseHeaders}
+                        data={courseJSON.classes}
+                        filename = {"CourseTable.csv"}>
```

Comment:
We should be able to remove the `{}` and the spaces around the `=` here.
```suggestion
                        filename="CourseTable.csv">
```